command/operator_debug.go

```diff
@@ -499,6 +519,20 @@ func (c *OperatorDebugCommand) collectPprof(path, id string, client *api.Client)
 	if err == nil {
 		c.writeBytes(path, "goroutine.prof", bs)
 	}
+
+	// Gather goroutine text output - debug type 1
```

Comment:
I know its a golang thing but can we add a small description to what the magic numbers mean?
```suggestion
	// Gather goroutine text output - debug type 1
	// debug type 1 writes the legacy text format for human readable output
```

Reply:
Good suggestion - fixed!